### DISCUSSION TITLE
Isn't the zip version meant to be truly portable? And not erase user data?

### DISCUSSION THREAD

**Discussion by mzso** [Alex313031/thorium#691] [Q&A]
Hi!
So I was hoping the zip version would work like a proper portable version and didn't dump any data like chromium browsers typically do.
I even see this at the patches section "Patches to enable truly Portable usage > Disable Encryption and Machine ID" and the relevant command line switches in THORIUM.BAT.

But when I started up thorium with the profile copied from a different chromium browser, with the exception of history and bookmarks all my data was erased without a notice.

Aren't the patches meant to prevent exactly this?

**Comment by Alex313031** [comment]
@mzso Fixed in this commit and will be live in the upcoming M126 release. > https://github.com/Alex313031/thorium/commit/88ec3acefe38469afe939281e7af394f495a9db7#diff-a93d0d958732c1a750ed0f1b76b6c34dd0990008919697612b94fa8350c23d92

**Reply by mzso** [reply]
Great! Thanks.
